tests/sync/session.cpp

```diff
@@ -196,6 +196,19 @@ TEST_CASE("SyncSession: management by SyncUser", "[sync]") {
         session = user->session_for_url(server.base_url() + path);
         CHECK(session);
     }
+
+    SECTION("a user cannot create multiple sessions for the same URL") {
```

Comment:
My thinking is that the test will fail when we remove the restriction, which should make it obvious that it needs to be removed.